packages/text/Text.tsx

```diff
@@ -3,8 +3,22 @@ import { TextStyle } from './TextStyles'
 import { TextProps } from './types'
 
 function Text(props: TextProps, ref?: ForwardedRef<HTMLParagraphElement>) {
-  const { size = 'md', color = 'default', ...rest } = props
-  return <TextStyle size={size} color={color} ref={ref} {...rest} />
+  const {
+    size = 'md',
+    weight = 400,
+    color = 'default',
+    strong,
+    ...rest
+  } = props
+  return (
+    <TextStyle
+      size={size}
+      weight={strong ? 700 : weight}
```

Comment:
Feels like this may be a bit confusing if you set `strong` and `weight` at the same time. Why not create another <StrongText> component, which would be just a wrapper for Text?

Reply:
Seems to be excessive and overwhelming to have two components with difference only font-weight prop.
`strong` prop still needed to provide backward-compatibility in cases that are already in use across our projects.

Reply:
Ah, I see, there were `strong` property before, forgot about that